docs/chapters/workflow/release/howto-release-pelux.rst

```diff
@@ -0,0 +1,98 @@
+* Decide on what Yocto release to follow. If this is a major release then
+  this is usually the latest Yocto release. If this is minor release, then
+  this was already decided when the major release was done, and you should
+  not change it.
+* Decide on what Qt version to support. The Qt project does not follow the
+  same release cadence as the Yocto project does. This will typically be the
+  latest Qt version on a major release, and the same as for the major
+  release in a minor release.
+
+Prepare the manifest
+^^^^^^^^^^^^^^^^^^^^
+* Prepare the layers
+
+  * Create a branch in each layer we maintain matching the Yocto release
+    that this PELUX release is targeting. This probably already exists, in
+    which case make sure that the branch is up-to-date.
+  * Create a tag in the branch pointing to the latest commit in the branch.
+
+* Create a release branch in pelux-manifests.
+* Point to specific layer revisions in ``pelux.xml`` and push the manifest to
+  the release branch.
+
+    * Don't point to tags here, since tags are moving targets. We create
+      tags in our own layers for convenience, and to make maintenance
+      easier, but don't point to them here.
+    * Make sure the commits pointed to are on the branches matching the
+      Yocto release we're targeting.
+    * For the Qt layers, point to revisions for the Qt release targeted in
+      this PELUX Baseline release.
+
+* Make sure the manifest builds properly in the CI system for all variants
+  without any unexpected warnings and with no errors.
+* Make sure each variant works on target as intended.
+* Create a tag in the release branch.
+
+Prepare the Software Factory Baseline
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+* Sync the Baseline with the Blueprint.
+* Make sure the docs are built properly by the CI system without warnings.
+* Proofread the generated docs.
+* Create a release branch, and push the latest docs to it.
+* Make sure the PELUX version is set in ``CMakeLists.txt``.
+* Create a tag in the release branch.
+* Make sure the CI system builds and deploys the release version of the docs to
+  ``pelux.io/software-factory/<VERSION>/``.
+
+Prepare the release notes
+^^^^^^^^^^^^^^^^^^^^^^^^^
+* Write down all the information from the steps done before touching any
+  files into the release notes.
+* Point out the branch, tag and commit for the manifest used in the release.
+* Point out the branch, tag and commit for the PELUX Software Factory
+  Baseline used in the release.
+* Mention that the release notes themselves are part of the release.
+
+Announce the release
+^^^^^^^^^^^^^^^^^^^^
+* Update the downloads page on ``pelux.io``. Put the release notes there.
+* Write a blog post on ``pelux.io`` announcing the release.
+* Post about the release to other relevant channels (mailing lists, social media
+  etc).
+
+After releasing
+^^^^^^^^^^^^^^^
+* Do a release retrospective with the team and improve this process. Document
```

Comment:
This note about the project work is pretty harmless but in general I don't think we should have any "working in the project" content in these docs. That's more of a project wiki kind of thing. I realize that it's a project specific decision though, but I'm pointing it out because this is our reference to users and we will recommend a split between "project work" and "platform/development work" that is defined in a way that will collide with this kind of content.

Reply:
Yes, perhaps the wording is a bit too much here. But I think it is an important part of the process to improve this process. Perhaps I can write that in a more generic way.

Reply:
Changed the wording now.